### ISSUE TITLE
DetailsList: High Contrast Mode has display errors

### ISSUE THREAD

**Issue by aaronyzhou** [OfficeDev/office-ui-fabric-react#2197]
### Bug Report

- __Package version(s)__: master
- __Browser and OS versions__: windows high contrast mode

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? (Yes, No) 

Requested priority: (Blocking, High, Normal, Low) normal

Products/sites affected: (if applicable) all Details List

#### Describe the issue:
1.
![image](https://user-images.githubusercontent.com/7392105/28140661-1b070b06-670e-11e7-9647-fc6a5882f5ed.png)
Gaps in header border
2;
the column seperator in HC  when resizing columns on details list is not visible
3. 
in High contrast,  line item's hovering is not accessible(without border or differentiated background color) in EDGE
4. screen goes black when resizing columns

**Comment by CmonsterC** [comment]
@kristoferbrown seems like this is not fixed except for the 1st point. why was it closed?

**Comment by kristoferbrown** [comment]
The remaining issues are resolved by PR #2589, looks like I prematurely closed this when creating that PR instead of resolving it.
